terraform/diagnostics.go

```diff
@@ -0,0 +1,17 @@
+package terraform
+
+import (
+	"log"
+
+	"github.com/hashicorp/hcl/v2"
+)
+
+func handleDiagnostics(issue string, diags hcl.Diagnostics, filename string) {
```

Comment:
This functionality will be used in the CLI, so I expect what will happen is wherever we use this function in this code it will log directly to the user's output. That is too much logging in my opinion and it will clutter the CLI output a lot

In the CLI we have debug logs for this sort of thing so we don't want to log directly. I think having debug logs would be nice but we need to figure this out a bit more before doing it. Do you think we should do this now or can we create a ticket so add debug logs and so think of a better solution for this?

Reply:
good point we didn't thought about that.
I think that creating a separate ticket to tackle debug logs sounds like a great idea.

Reply:
Ah nice! @teodora-sandu sounds like you have some thoughts already, can you open a ticket about this?